sormas-api/src/main/java/de/symeda/sormas/api/feature/FeatureType.java

```diff
@@ -45,7 +45,7 @@
 			SAMPLES_LAB }),
 	TASK_MANAGEMENT(true, true, null),
 	WEEKLY_REPORTING(true, true, null),
-	IMMUNIZATION_MANAGEMENT(true, true, null),
+	IMMUNIZATION_MANAGEMENT(true, true, null, FeatureTypeProperty.REDUCED),
```

Comment:
Please extend FeatureService.createMissingFeatureConfigurations to also persist the default feature type properties. Otherwise it would be inconsistent (creating entries with the default enabled values of feature configurations in the database, but not including the default values for feature type properties).